Sources/Apodini/Properties/Request.swift

```diff
@@ -25,30 +25,6 @@ protocol Request: CustomStringConvertible, CustomDebugStringConvertible {
     func retrieveParameter<Element: Codable>(_ parameter: Parameter<Element>) throws -> Element
 }
 
-@propertyWrapper
-// swiftlint:disable:next type_name
-struct _Request: Property, RequestInjectable {
-    private var request: Request?
-    
-    
-    var wrappedValue: Request {
-        guard let request = request else {
-            fatalError("You can only access the request while you handle a request")
-        }
-        
-        return request
-    }
-    
-    
-    init() { }
-
-
-    mutating func inject(using request: Request) throws {
-        self.request = request
-    }
-}
```

Comment:
I think it is still useful to access the `Request` in a Handler to get the current eventloop. My idea was it to use the current `Request` for scheduling of `Job`s in a Handler. Perhaps we can add `Request` to `dynamicValues` in order to access it with `@Environment` in a Handler.

Reply:
@awoc I agree that having access to the `EventLoop` is important for various features. When working on the PR I already created an issue (#93) that documents this problem. As `Request` is an internal type and `_Request` was a left over from the very early days where we were using `Vapor.Request` and all the other functionality except the `eventLoop` (e.g. the `Endpoint`) shouldn't normally be accessible within a `Handler`.
I like the idea of accessing the `EventLoop` using the `Environment` in a non global manner as it is injected with every execution of a handler but would oppose injecting the whole `Request`.

Reply:
My bad, I didn't see the open issue

Reply:
No problem :)
Would an access to an EventLoop solve your problem or should we consider some sort of access to a Request type anyways?

Reply:
Using only the EventLoop is totally sufficient 👍